tools/warning-to-plist/tests/unit/clang_sanitizer_output_test_files/files/ubsan2.cpp

```diff
@@ -0,0 +1,31 @@
+#include <limits.h>
+
+// Load of a value of an enumerated type which is not in the range of
+// representable values for that enumerated type.
+int f()
+{
+  enum E {
+    a = 1
+  };
+
+  int value = 7;
+  enum E *e = (enum E *)&value;
+  return *e;
+}
+
+// Load of a bool value which is neither true nor false.
+bool g()
+{
+  int result = 2;
+  bool *predicate = (bool *)&result;
+  if (*predicate) { // Error: variable is not a valid Boolean
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Condition '*predicate' is always true](https://app.codacy.com/app/CodeChecker/codechecker/pullRequest?prid=4169881)